### ISSUE TITLE
Projects: cannot fund multiple issues at once

### ISSUE THREAD

**Issue by chadoh** [AutarkLabs/open-enterprise#1539]
Reported by @philh:

> bumping into issues when trying to batch-fund issues. No exception, but it's impossible to fund more than 1 issue at a time

**Comment by chadoh** [comment]
@philh I just accomplished funding multiple issues at once at my Rinkeby org, https://rinkeby.aragon.org/#/chadoh20191105

It was a little confusing; I almost missed that the funding request got forwarded to the Voting app, and I had to click over there, vote Yes, wait for voting to close, and then enact the vote. But once I did, both of my issues received their funding:

<img width="730" alt="rinkeby funding multiple issues at once" src="https://user-images.githubusercontent.com/221614/68223240-6e3cf780-ffba-11e9-9cf6-062bd6576859.png">

Do you observe this same behavior?

**Comment by PhilH** [comment]
I did succeed to fund some of the issues, either through voting when a vote is needed, or just after submitting the panel when the permission allows me to fund without a vote (I did both while I was playing with the Permissions system). But then I also got this "transaction impossible" message when attempting to fund issues.

Now I suspect that it's an issue with the app losing the connection with metamask or something wrong with metamask itself, maybe a timeout. I started another test which went fine, and then after a little while I bumped into the problem again. After reloading everything, it worked. So I assume that this is not an OE problem. The issue should be closed.
